explorer.go

```diff
@@ -31,6 +32,25 @@ const (
 	addressRows     = 2000
 )
 
+type ByValue []dcrjson.TxRawResult
```

Comment:
In the event that we want other things in this package sorted by value, this naming will turn out to be too general.  Just an unfortunately consequence of go's lack of generics.  Anyway, if implementing `sort.Interface` is the chosen solution, I would go with something like `TxRawByValue`.

However, since `Len` and `Swap` are completely boilerplate here, this is a prime case to utilize the new [`sort.Slice`](https://golang.org/pkg/sort/#Slice) function.  For example:

```go
func voutTotal(vouts []dcrjson.Vout) (total float64) {
	for i := range vouts {
		total += vouts[i].Value
	}
	return
}

sort.Slice(data.RawTx, func(i, j int) bool {
	return voutTotal(data.RawTx[i].Vout) > voutTotal(data.RawTx[j].Vout)
})
```

But I leave that up to you.